benchmarks/hash_table/dynamic_map_bench.cu

```diff
@@ -86,7 +86,8 @@ static void BM_dynamic_insert(::benchmark::State& state)
 
   std::size_t batch_size = 1E6;
   for (auto _ : state) {
-    map_type map{initial_size, -1, -1};
+    map_type map{
+      initial_size, cuco::sentinel::empty_key<Key>{-1}, cuco::sentinel::empty_value<Value>{-1}};
```

Comment:
Not for this PR, but looking at this PR makes me feel like the `sentinel::` namespace is unnecessary and just bloats the line length. It was probably my idea to begin with, but now I'm regretting it :sweat_smile:

Reply:
The code is more readable though.